datumaro/plugins/camvid_format.py

```diff
@@ -159,13 +160,17 @@ def _load_items(self, path):
         items = {}
         with open(path, encoding='utf-8') as f:
             for line in f:
-                objects = line.split()
+                search = CamvidPath.PATTERN.search(line.strip('\n'))
```

Comment:
Why we strip only `\n`?

Reply:
We must always have a space after the first path. In [test without masks](https://github.com/openvinotoolkit/datumaro/blob/944e9f9f8ae2b10aea5eff9573e2de65844538c0/tests/test_camvid_format.py#L148), after `strip()`, we get the `test/a/b/1.png` without a space. As a result, search returns None.

Reply:
There is no such requirement. But I'm having difficulty with a regex that supports a string without a space. If we replace `+` with `*` in the regex (`r'(.+\.\S+)(?:\s*(.+\.\S+)?)\s*'`), we cannot find two paths.

Reply:
Remove the last `\s*`

Reply:
This doesn't work as everything is recorded in the first group.

Reply:
Did you try `(.+\.\S+)(?:\s+(.+\.\S+)?)?` ?

Reply:
Yes, also everything is recorded in one group

Reply:
If we don't have a mask, it is what we need to obtain, isn't it?

Reply:
Yes. But everything is recorded in one group even we have a mask.

Reply:
How about eager capture in the first group then? `(.+?\.\S+)(?:\s+(.+\.\S+)?)?`

Reply:
It works, thanks!